app.py

```diff
@@ -30,6 +34,8 @@
 from poprox_concepts.domain.topics import GENERAL_TOPICS
 from poprox_concepts.internals import from_hashed_base64
 
+COMPENSATION_OPTIONS = COMPENSATION_CARD_OPTIONS + COMPENSATION_CHARITY_OPTIONS + ["Decline payment"]
```

Comment:
I'm stuck on the relative PRO/CON of including `Decline payment` as one of the listed options in https://github.com/CCRI-POPROX/poprox-concepts/pull/35 , leaving things as they are where there's one option not documented there, or requesting that we make the field null if payment is declined.

I think it's probably fine to deploy as-is, but if feels _weird_ to have the full set of string-values split over two files.

Reply:
Yeah, I don't really like this either but it also didn't seem worth a full third category. I guess it could be in something like `OTHER_COMPENSATION_OPTIONS`?